doc/setup/install.rst

```diff
@@ -475,6 +475,14 @@ Enterprise Edition: (see :ref:`setup/install/editions` to get access)
 
   $ git clone https://github.com/odoo/enterprise.git
 
+Alternaly, if you only wish to fetch one specific version:
+
+.. code-block:: console
```

Comment:
I am not sure about this part. People don't like when they got too many options, they like one universal way. The issue with `--depth 1` is that it is more difficult to update. Instead, maybe add a message explaining odoo repository is quite heavy and for lighter system, they can use `--depth 1` and `--single-branch` pointing [to the documentation](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---no-single-branch)